internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -168,6 +171,12 @@ func (*Sequencer) Install(r runtime.Runtime) []runtime.Phase {
 			).Append(
 				"install",
 				Install,
+			).Append(
+				"loadMeta",
```

Comment:
```suggestion
				"meta",
```

as used everywhere else

Reply:
fixed